### ISSUE TITLE
Duplicated log messages

### ISSUE THREAD

**Issue by jrziviani** [kimchi-project/wok#30]
When creating a new process for websockify, both process (main and child) is probably using the same log handler. The problem is that all log entries are being duplicated:

```bash
% ps -efa | grep python
root      5204  5196 21 22:53 pts/17   00:00:01 python2 src/wokd --host=0.0.0.0
root      5235  5204  0 22:53 pts/17   00:00:00 python2 src/wokd --host=0.0.0.0
```

```
% tail -f log/wok-error.log | grep -v WOKOBJST0001E
KCHNET0011E: Network name must be a string without slashes (/) or quotes (")
KCHNET0011E: Network name must be a string without slashes (/) or quotes (")
```

**Comment by jrziviani** [comment]
Please, can you assign this one to me? I'll investigate this issue.

**Comment by alinefm** [comment]
Fixed by cf6f28e8ae19c60d780dbb730f2431d17360a8f8
